docs/_posts/1871-05-10-Louis-Rau.md

```diff
@@ -20,17 +20,17 @@ Meine Lieben Alle,
 Umstehenden Bf von <span data-lang="fr">Guillier</span> habe geöffnet
 weil ich fürchtete, es sei nicht <abbr title="ou 'wieso'?">nahe</abbr> in <span data-lang="fr">Paris</span>. Einstweilen
 schaut es heute recht gut für uns Juden in <span data-lang="fr">Paris</span> aus; die
-<span data-lang="fr">Communten</span> haben eine grosse Angst & so glaube ich
+<span data-lang="fr">Communi[s]ten</span> haben eine grosse Angst & so glaube ich
```

Comment:
Are you sure this can't be an attempt to talk about the Communards (now called ["Kommunarden"](https://de.wikipedia.org/wiki/Pariser_Kommune) in German nowadays, but the word might not have existed at the time so maybe he tried to make it up from the French one?)

Reply:
Quite sure. There are two indications:
- there’s no doubt about the third to last letter being a `t` rather than a `d`. Ludwig quite consequently applies the horizontal bar on the `t`.
- Ludwig is writing in Latin script here, so the `a` would be (more or less) round and closed like in the word `Paris` in the line below. Also, if it were a German script `a`, there would be two dents missing instead of only one.